File: DSA PRACTICE/LECTURE8/SwitchProblem.java
// SwitchProblem:01

public class SwitchProblem
{
	public static void main(String[] args) 
	{
		int n=1;
		switch (n) 
		{
			case 1: System.out.print("Number First");
					break;
			case 2: System.out.print("Number Second");
					break;
			case '1' : System.out.print("Character First");
					break;
			// case "a" : System.out.print("String First");  error
			// 		break;
			default: System.out.print("Not match");	
		}
	}
}

/*
Number First
*/

File: DSA PRACTICE/LECTURE8/SwitchProblem.cpp
// SwitchProblem:01
#include<iostream>
using namespace std;

int main()
{
	int n=1;
	switch(n*2)
	{
		case 1: cout<<"Number First";
					break;
		case 2: cout<<"Number Second";
					break;
		case '1' : cout<<"Character First";
					break;
		// case "a" : cout<<"String First"; // invalid conversion
		// // 		break;
		default: cout<<"Not match";	
	}
    return 0;
}

/*
Number Second
*/